src/components/Header/styles.js

```diff
@@ -3,16 +3,22 @@ import styled from 'styled-components';
 export const Container = styled.header`
   position: fixed;
   width:100vw;
-  padding: ${({theme})=> theme.scale.base*2}px;
+  padding: ${({theme})=> theme.scale.base}px;
   background-color: ${({theme})=>theme.palette.primary};
   display:flex;
   align-items: center;
   justify-content: space-between;
+
+  @media (max-width:484px) {
```

Comment:
try using the value of `isMobile` given by the useMediaQuery custom hook on the parent element